test/BaselineOutput/Common/MulticlassLogisticRegression/MulticlassLogisticRegression-CV-iris-out.txt

```diff
@@ -39,6 +43,10 @@ Accuracy(micro-avg): 0.985915
 Accuracy(macro-avg): 0.983333
 Log-loss:           0.075812
 Log-loss reduction: 0.930169
+Top K accuracy(All K)K=1: 0.985915
+Top K accuracy(All K)K=2: 1.000000
+Top K accuracy(All K)K=3: 1.000000
+Top K accuracy(All K)K=4: 1.000000
```

Comment:
I might recommend formatting as either:
* Top-1 accuracy: 0.985915<br/>Top-2 accuracy: 1.000000<br/>...

or perhaps to match the [NDCG@N format](https://github.com/dotnet/machinelearning/blob/db4ecc0135baffc8930201a85fb3e9a101cdfa46/test/BaselineOutput/Common/Command/CommandTrainRanking-5-out.txt#L20-L25).

* Top accuracy@1: 0.985915<br/>Top accuracy@2: 1.000000<br/>...

Also, this list will get quite long if there are thousands of classes (which does happen).

Reply:
If having thousands of classes is a possibility, should we instead look into using the `TopKAccuracy` parameter to limit how many K-s we include in the TopKAccuracyForAllK array?

Reply:
changed the output to 
-top-1-accuracy: 0. nnnn
-top-2-accuracy: 0.nnn
which matches how it used to output for the one and only top-n-accuracy:

Reply:
Here is a snippet from one of the outputs, I don't think it's visible in any of the test files any more because the tests don't specify a top K parameter value and we now respect that parameter